runner/src/server/plugins/engine/pluginHandlers/files/prehandlers/handleUpload.ts

```diff
@@ -44,16 +44,13 @@ export async function handleUpload(
     }
 
     let response;
-
+    const errors = new Set();
     try {
       response = await uploadService.uploadDocuments(streams);
     } catch (err) {
       if (err.data?.res) {
-        const { error } = uploadService.parsedDocumentUploadResponse(err.data);
-        request.pre.errors = [
-          ...request.pre.errors,
-          parsedError(fieldName, error),
-        ];
+        response = uploadService.parsedDocumentUploadResponse(err.data);
```

Comment:
response should be the response (rather than a parsed error). I think you had this a few diffs ago